src/services/findAllReferences.ts

```diff
@@ -1105,29 +1105,30 @@ namespace ts.FindAllReferences.Core {
         }
 
         // If we got a type reference, try and see if the reference applies to any expressions that can implement an interface
-        const containingTypeReference = getContainingTypeReference(refNode);
-        if (containingTypeReference && state.markSeenContainingTypeReference(containingTypeReference)) {
-            const parent = containingTypeReference.parent;
-            if (hasType(parent) && parent.type === containingTypeReference && hasInitializer(parent) && isImplementationExpression(parent.initializer)) {

```

Comment:
Dont you still need some kind of equivalent for ```parent.type === containingTypeReference``` otherwise doing this on anything eg node is initializer but parent has type would pass through this? (May be its some form of assert?)